apps/web/app/globals.css

```
@@ -3812,6 +3892,116 @@ button {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5px;
+  padding: 0 6px;
+}
+
+.themeChoicePreviewBody i {
+  height: 3px;
+  border-radius: 2px;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)